frontend/src/pages/projects/screens/detail/storage/ManageStorageModal.tsx

```diff
@@ -79,6 +87,12 @@ const ManageStorageModal: React.FC<AddStorageModalProps> = ({ existingData, isOp
 
     if (existingData) {
       const pvcName = existingData.metadata.name;
+      if (getPvcDisplayName(existingData) !== createData.nameDesc.name) {
+        await updatePvcDisplayName(pvcName, namespace, createData.nameDesc.name);
+      }
+      if (getPvcDescription(existingData) !== createData.nameDesc.description) {
+        await updatePvcDescription(pvcName, namespace, createData.nameDesc.description);
+      }
```

Comment:
If they are not dependant, you should always allow async be async... no sense in holding up the rest of the execution

Reply:
I tried to async update this before but looks like it will not immediately get the new name so sometimes the PVC name in the list will not refresh after you update it, I don't know what happened, and that's why I added this to hold until it really updates

Reply:
If you truly need to wait on two operations, Promise.all is how you do it... but in this case, it's two patches to the same object, you could have made it 1 call.